packages/server/src/__tests__/plugin/cacheControl/collectCacheControlHints.ts

```diff
@@ -41,11 +41,11 @@ export async function collectCacheControlHintsAndPolicyIfCacheable(
   const response = await server.executeOperation({ query: source });
   await server.stop();
 
-  expect(response.errors).toBeUndefined();
+  expect(response.result.errors).toBeUndefined();
 
   return {
     hints: cacheHints,
-    policyIfCacheable: response.extensions!.__policyIfCacheable__,
+    policyIfCacheable: response.result.extensions!.__policyIfCacheable__ as any,
```

Comment:
There's a type argument to `FormattedExecutionResult` that might be the solution to the `as any` here

Reply:
hmm, not really sure where that would work.  we would want to add generics to `GraphQLResponse` too then? which would have to be different for different operations?  or are you just saying add a slightly different cast here?

note that the reason for this change is just that the old typings had `extensions` have `any` as its record value type and now it's `unknown`.